bin/deps

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+docker-compose -f docker-compose-deps.yml up
```

Comment:
It would be nice to get away from multiple compose files.

Does something like `docker-compose up clamav postgres redis` not work here?